proxy/src/main/java/com/velocitypowered/proxy/console/VelocityConsole.java

```diff
@@ -56,8 +55,7 @@ public final class VelocityConsole extends SimpleTerminalConsole implements Cons
   private PermissionFunction permissionFunction = ALWAYS_TRUE;
   private final @NotNull Pointers pointers = ConsoleCommandSource.super.pointers().toBuilder()
           .withDynamic(PermissionChecker.POINTER, this::getPermissionChecker)
-          .withDynamic(Identity.LOCALE, () -> ClosestLocaleMatcher.INSTANCE
-              .lookupClosest(Locale.getDefault()))
+          .withDynamic(Identity.LOCALE, Locale::getDefault)
```

Comment:
I think it would have been ideal to keep this, since the default language and the language in which messages are sent to the console could vary, while the language defined in the JVM could always be checked with a `Locale.getDefault()`

Reply:
Do you mean just in the `pointers` or in the `sendMessage` method too?

In the first case, I would find this weird to have a mismatch between these two.

In the second case, I think it the same problem as for players, translators will have no way to know the original locale of the console. Sure it is `Locale.getDefault()` but translators don't know if they are translating something for the console or for a player.

Reply:
I am referring to the console pointers, since anyone can use `Locale.getDefault()`, but the language in which the messages are sent to the console may vary.
But it is something without much importance and it can be left as you have done it